.github/workflows/main.yml

```diff
@@ -19,13 +19,13 @@ jobs:
       matrix:
         platform:
           # macos-x64
-          - macos-latest
+          - macos-12
           # macos-aarch64
           - macos-14
           # linux-x64
           - ubuntu-20.04
           # windows-x64
-          - windows-latest
+          - windows-2019
```

Comment:
Following the ubuntu logic of "older versions might have wider support???"

This seems to work fine on my up-to-date Windows 11 install